core/trino-main/src/main/java/io/trino/dispatcher/DispatchManager.java

```diff
@@ -146,7 +146,7 @@ public ListenableFuture<Void> createQuery(QueryId queryId, Span querySpan, Slug
         requireNonNull(sessionContext, "sessionContext is null");
```

Comment:
Please reduce commit message width
"withing history" typo

Reply:
Commit title is about readability but the message is about perf overhead. Please choose one of those two motivations, whichever is actually impactful, for commit title and message.